Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM freesurfer/freesurfer:7.1.1 as base
+
+LABEL maintainer="support@flywheel.io"
+
+RUN (curl -sL https://rpm.nodesource.com/setup_12.x | bash -) \
+  && yum clean all -y \
+  && yum update -y \
+  && yum install -y zip unzip nodejs tree libXt libXext ncurses-compat-libs \
+  && yum clean all -y \
+  && npm install npm --global
+
+RUN npm install -g bids-validator@1.5.4
+
+RUN source $FREESURFER_HOME/SetUpFreeSurfer.sh
+
+# extra segmentations require matlab compiled runtime
+RUN fs_install_mcr R2014b
+
+# Save docker environ
+ENV PYTHONUNBUFFERED 1
+RUN python -c 'import os, json; f = open("/tmp/gear_environ.json", "w"); json.dump(dict(os.environ), f)'
```

Comment:
No, but please check my logic on doing it here instead.  Freesurfer needs python3.6 which is installed in `/usr/local/freesurfer/python`.  I want to use a more recent version of python to run `run.py`.  So I save the environment and then install the newer version of python.  Running the tests inside the gear uses the newer version of python but any call to a freesurfer executable does not have the newer python in its path so it does whatever recon-all does using the environment saved here.  It does seem to work but am I thinking about this corectly?

Reply:
woof.  seems complicated.  I would get another set of eyes to look at this (Nicolas or Nate maybe?).  I have vague misgivings about this but I can't put my finger on exactly what or why.

Reply:
I started down the path of using Freesurfer's python3.6 but I was starting to make a lot of changes.  I had to install pip there to install flywheel and then I had to modify the test script to use that python.  Then I got to thinking about how it is probably a good idea to keep Flywheel stuff separate from the Freesurfer python because somehow somewhere mixing them might cause Freesurfer to fail in a hard to understand way.  So I'm going with leaving a comment in the Dockerfile to say what's going on.